scripts/controls.js

```diff
@@ -79,15 +79,15 @@ window.onmousewheel = function(e) {
 	} else {
 	  mouse.current.x = getMousePos(e)[0];
 	  mouse.current.y = getMousePos(e)[1];
-	  
-	  cropx = (cropx + mouse.current.x * (1 + zoomAmount)) - mouse.current.x * (zoomLevel - (zoomLevel - 1));
```

Comment:
How did I not realize that this was just multiplying by 1?

Reply:
Haha, sometimes you are thinking about so many things at once, you don't realize what you're doing. It happens to the best of us :)